tests/integration/express-server.spec.ts

```diff
@@ -139,6 +139,19 @@ describe('Express Server', () => {
     });
   });
 
+  it('should reject invalid additionalProperties', () => {
```

Comment:
Can you c/p this to the other integration tests? We should make sure this works regardless of framework / additionalProperties Setting.

Reply:
Yes, it's done